src/main/java/seedu/address/ui/PersonDetailedView.java

```diff
@@ -0,0 +1,67 @@
+package seedu.address.ui;
+
+import java.util.Comparator;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.FlowPane;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.Region;
+import seedu.address.model.person.Person;
+
+
+
+/**
+ * A UI Component that displays a detailed view of a {@code Person}
+ */
+public class PersonDetailedView extends UiPart<Region> {
+    private static final String FXML = "PersonDetailedView.fxml";
+
+    public final Person person;
+    @FXML
+    private HBox cardPane;
+    @FXML
+    private Label name;
+    @FXML
+    private Label id;
+    @FXML
+    private Label phone;
+    @FXML
+    private Label address;
+    @FXML
+    private Label email;
+    @FXML
+    private Label birthday;
+    @FXML
+    private Label hasPaid;
+    @FXML
+    private FlowPane tags;
+    @FXML
+    private Label placeholderLabel;
+    @FXML
+    private ImageView profileImage;
+
+    /**
+     * Creates a {@code PersonCode} with the given {@code Person} to display.
+     */
+    public PersonDetailedView(Person person) {
+        super(FXML);
+        this.person = person;
+
+        Image profileImg = new Image(getClass().getResourceAsStream("/images/profilepicture.png"));
```

Comment:
Noted that the profileImg for each contact is currently the same. Let's try to revise this in the next iteration alongside functionality to upload .png files for each person!